src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -137,6 +137,21 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
                         deployment.Annotations.Add("dapr.io/enabled", "true");
                         deployment.Annotations.Add("dapr.io/id", project.Name);
                         deployment.Annotations.Add("dapr.io/port", (httpBinding.Port ?? 80).ToString(CultureInfo.InvariantCulture));
+
```

Comment:
Do you have any interest in making this work for local dev as well? You could add the same thing to the command line here: https://github.com/dotnet/tye/pull/298/files#diff-e1a701ef2a3ac88566fe0e681b1e224fR40

If you don't feel like doing this I'll do it in a follow-up after this PR is merged.

Reply:
Could you clarify more? I don't get the point of `making this work for local dev`

Reply:
Please help me with it. Thank you.

Reply:
local dev => `tye run`. You'd need to pass the commandline arguments to `daprd` - ex: `-log-level info -config myConfig`

Reply:
Did that. Please check it